step-templates/tomcat-undeploy-application.json

```diff
@@ -0,0 +1,73 @@
+    },
+    "Parameters": [
+      {
+        "Id": "a3a0b5b5-067f-40fe-b93d-833bbe901937",
+        "Name": "UndeployManagerUrl",
+        "Label": "Tomcat Manager URL",
+        "HelpText": "This is the URL of the Tomcat Manager instance that the package will be uploaded to. This URL is relative to the host that is executing the step. So if the Tentacle is on the same host as Tomcat, then this value may be something like http://localhost:8080/manager.",
+        "DefaultValue": "http://localhost:8080/manager",
+        "DisplaySettings": {
+          "Octopus.ControlType": "SingleLineText"
+        }
+      },
+      {
+        "Id": "bb902de7-ad45-4f76-8111-0d080d491c77",
+        "Name": "UndeployManagementUser",
+        "Label": "Management User",
+        "HelpText": "The username to use with the management interface. This user must be assigned to the manager-script group in the tomcat-users.xml file.",
+        "DefaultValue": "",
+        "DisplaySettings": {
+          "Octopus.ControlType": "SingleLineText"
+        }
+      },
+      {
+        "Id": "7be02a9f-bc24-4df3-bf2b-3d4963c1d6ae",
+        "Name": "UndeployManagementPassword",
```

Comment:
I think it would be helpful to namespace these parameters. Maybe some like _Tomcat.Undeploy.ManagementPassword_.